keydata.go

```diff
@@ -50,7 +50,7 @@ const (
 )
 
 var (
-	keyDataVersion        int = 2
+	keyDataGeneration     int = 2
```

Comment:
I've actually got a follow-up PR that makes this public (so that the version can be bound into the AAD)

Reply:
oh sorry about that, I thought I had that change in here already, I also make it public in the final PR because I needed it for a test for AAD https://github.com/snapcore/secboot/pull/271/commits/33445cf5c8ba0ebb5e609a0af9e9122232d8f5e9